<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SfoEditorDialog</class>
 <widget class="QDialog" name="SfoEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PARAM.SFO Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>PARAM.SFO settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="titleIDLabel">
          <property name="text">
           <string>Title_ID :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="titleIDLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="versionLabel">
          <property name="text">
           <string>Version :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="versionLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string> Category :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="categoryComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>DG - Disc Game</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DM - Disc Movie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DP - Disc Package</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GD - Game Data</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HG - Hard disk Game</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IP - Install Package</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SD - Save Data</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="bootableLabel">
          <property name="text">
           <string>Bootable :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="bootableComboBox">
            <item>
             <property name="text">
              <string>yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>no</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="parentalLevelLabel">
            <property name="text">
             <string>Parental Level :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="parentalLevelComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>0 - none</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11 - full</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="appVerLabel">
          <property name="text">
           <string>App Ver :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="appVerLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="ps3SystemVerLabel">
            <property name="text">
             <string>PS3 System Ver :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ps3SystemVerLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="resolutionLabel">
          <property name="text">
           <string>Resolution :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="res1080CheckBox">
            <property name="text">
             <string>1080</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="res720CheckBox">
            <property name="text">
             <string>720</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="res576169CheckBox">
            <property name="text">
             <string>576 (16:9)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="res480169CheckBox">
            <property name="text">
             <string>480 (16:9)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="res576CheckBox">
            <property name="text">
             <string>576</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="res480CheckBox">
            <property name="text">
             <string>480</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="soundFormatLabel">
          <property name="text">
           <string>Sound Format :</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="sfDolbyCheckBox">
            <property name="text">
             <string>Dolby Digital</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="sfDTSCheckBox">
            <property name="text">
             <string>DTS Digital Surround</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="sfLPCM5CheckBox">
            <property name="text">
             <string>5.1 LPCM</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="sfLPCM2CheckBox">
            <property name="text">
             <string>2 LPCM</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="sfLPCM7CheckBox">
            <property name="text">
             <string>7.1 LPCM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="resetPushButton">
       <property name="text">
        <string>&amp;Reset Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="discardPushButton">
       <property name="text">
        <string>&amp;Discard Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>&amp;Save Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
